docs/TelemetryUnit_ESP8266.md

```diff
@@ -0,0 +1,94 @@
+# Telemetry unit (ESP8266)  
+
+As we know **Telemetry** is an automated communications process by which measurements and other data is collected at remote or inaccessible points and transmitted to receiving equipment for monitoring. 
+
+In this case, we are using the ESP8266 as a telemetry unit. We collect the data from the logger (Arduino MEGA) and the ESP8266 will be in charge of sending the collected data to Ubidots. 
+
+## Hardware setup - Wiring
+
+It is important identify carefully the PINOUT of the ESP8266, because the ESP8266 can break if you make a wrong connection.
+
+![ESP8266 connection](https://raw.githubusercontent.com/guyz/pyesp8266/master/esp8266_pinout.png)
+
+To program the **ESP8266** you have to use a programmer as **UartSBee**, also you can use the **Arduino MEGA** to program the ESP8266. To upload the code into the ESP8266 you have to follow the connections below.
```

Comment:
I think is better giving them both solution, because most of the user who ask about this library is because they want a cheaper product. So, they just need ESP + Arduino to start with it. What do you think? 

As you can see on the document, I tried to make the explanation easier to understand, adding diagrams, and troubleshootings